build.gradle.kts

```diff
@@ -0,0 +1,188 @@
+
+    // Compat fixes
+    modCompileOnly(fabricApi.module("fabric-renderer-indigo", project.property("fapi_version").toString()))
+    modCompileOnly("maven.modrinth:sodium:${project.property("sodium_version")}") { isTransitive = false }
+    modCompileOnly("maven.modrinth:lithium:${project.property("lithium_version")}") { isTransitive = false }
+    modCompileOnly("maven.modrinth:iris:${project.property("iris_version")}") { isTransitive = false }
+    modCompileOnly("maven.modrinth:indium:${project.property("indium_version")}") { isTransitive = false }
+    modCompileOnly("de.florianmichael:ViaFabricPlus:${project.property("viafabricplus_version")}") { isTransitive = false }
+
+    // Baritone
+    modCompileOnly("meteordevelopment:baritone:${project.property("baritone_version")}")
+
+    // Libraries
+
+    implementation("meteordevelopment:orbit:${project.property("orbit_version")}")
+    shadow("meteordevelopment:orbit:${project.property("orbit_version")}")
+
+    implementation("meteordevelopment:starscript:${project.property("starscript_version")}")
+    shadow("meteordevelopment:starscript:${project.property("starscript_version")}")
+
+    implementation("meteordevelopment:discord-ipc:${project.property("discordipc_version")}")
+    shadow("meteordevelopment:discord-ipc:${project.property("discordipc_version")}")
+
+    implementation("org.reflections:reflections:${project.property("reflections_version")}")
+    shadow("org.reflections:reflections:${project.property("reflections_version")}")
+
+    implementation("io.netty:netty-handler-proxy:${project.property("netty_version")}") { isTransitive = false }
+    shadow("io.netty:netty-handler-proxy:${project.property("netty_version")}") { isTransitive = false }
+
+    implementation("io.netty:netty-codec-socks:${project.property("netty_version")}") { isTransitive = false }
+    shadow("io.netty:netty-codec-socks:${project.property("netty_version")}") { isTransitive = false }
+
+    implementation("de.florianmichael:WaybackAuthLib:${project.property("waybackauthlib_version")}")
+    shadow("de.florianmichael:WaybackAuthLib:${project.property("waybackauthlib_version")}")
+
+    // Launch sub project
+    shadow(project(":launch"))
+}
+
+loom {
+    accessWidenerPath = file("src/main/resources/meteor-client.accesswidener")
+}
+
+afterEvaluate {
+    tasks.migrateMappings.configure {
+        setOutputDir("src/main/java")
+    }
+}
+
+tasks {
+    processResources {
+        val propertiesMap = mapOf(
+            "version" to project.version,
+            "devbuild" to devBuild,
+            "commit" to commit,
+            "minecraft_version" to project.property("minecraft_version"),
+            "loader_version" to project.property("loader_version")
+        )
+
+        inputs.properties(propertiesMap)
+        filesMatching("fabric.mod.json") {
+            expand(propertiesMap)
+        }
+    }
+
+    jar {
+        from("LICENSE") {
+            rename{
```

Comment:
add a space (`rename {`) and make into single line, like in the `shadowJar` task